test/query_test.exs

```diff
@@ -0,0 +1,40 @@
+defmodule QueryTest do
+  use ExUnit.Case
+
+  setup_all do
+    Application.put_env(:rethinkdb_ecto_test, TestRepo, [])
+
+    {:ok, conn} = RethinkDB.Connection.start_link
+    {:ok, _} = TestRepo.start_link
+    RethinkDB.Query.table_create("posts") |> RethinkDB.Connection.run(conn)
+
+    {:ok, conn: conn}
+  end
+
+  test "get, insert and update queries work" do
+    {:ok, test_model} = TestRepo.insert(%TestModel{title: "yay"})
+    from_db = TestRepo.get(TestModel, test_model.id)
+    assert test_model == from_db
+
+    # update_changeset = TestRepo.changeset(from_db, %{title: "yayay"})
```

Comment:
Authorized on travis. I'll take a look tonight and see what's going on.

Thank you for going above and beyond on this. This repo was a quick hack one afternoon and I've been pretty blindsided by its popularity. I'll feel a lot more comfortable with people using this with these basic tests in place. It also makes it easier for people to get involved.